tests/test_polyMap.py

```diff
@@ -274,6 +274,34 @@ def test_PolyMapPolyMapUnivertible(self):
             # so no inverse is possible
             pm.polyTran(False, 1e-3, 1e-3, 10, [-1.0], [2.5])
 
+    def test_PolyMapDM10496(self):
+        """Test for a segfault when simplifying a SeriesMap
+
+        We saw an intermittent segfault when simplifying a SeriesMap
+        consisting of the inverse of PolyMap with 2 inputs and one output
+        followed by its inverse (which should simplify to a UnitMap
+        with one input and one output). David Berry fixed this bug in AST
+        2017-05-10.
+
+        I tried this test on an older version of astshim and found that it
+        triggering a segfault nearly every time.
+        """
+        coeff_f = np.array([
+            [-1.1, 1, 2, 0],
+            [1.3, 1, 3, 1],
+        ])
+        coeff_i = np.array([
+            [1.6, 1, 3],
+            [-3.6, 2, 1],
+        ])
+
+        for i in range(1000):
```

Comment:
Please document why there's a big for loop here, and lift the assert into the loop (might as well check each of the returns, incase the bug manifests itself as data corruption instead of a segfault).